docs/guides/private-submodules.rst

```diff
@@ -0,0 +1,92 @@
+Using Private Git Submodules with Read the Docs for Business
+============================================================
+
+Read the Docs uses SSH keys (with read only permissions) in order to clone private repositories.
+An SSH key is automatically generated and added to your main repository, but not on your submodules.
+In order to give Read the Docs access to clone your submodules you'll need to add the public SSH key to each repository of your submodules.
+
+.. note::
+
+   You can manage which submodules Read the Docs should clone using a configuration file.
+   See :ref:`config-file/v2:submodules`.
+
+Project's SSH Key
+-----------------
+
+You can find the public SSH key of your Read the Docs project by
+
+- Going to the :guilabel:`Admin` tab of your project
+- Click on ``SSH Keys``
+- Click on the fingerprint of the SSH key (it looks like ``6d:ca:6d:ca:6d:ca:6d:ca``).
+
+.. note::
+   
+   The private part of the SSH key is kept secret.
+
+GitHub
+------
+
+For GitHub, Read the Docs uses `deploy keys with read only access <https://developer.github.com/v3/guides/managing-deploy-keys/#deploy-keys>`__.
```

Comment:
```suggestion
```

I'd remove this since it does not add value for the user following this guide, IMO

Reply:
@stsewd ping

Reply:
I think this does add value, users can verify what we are telling them is true and find more information about the configuration by themselves.